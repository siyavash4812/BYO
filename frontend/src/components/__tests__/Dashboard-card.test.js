import React from "react";
import renderer from "react-test-renderer";

import DashboardCard from "../Dashboard-card";

describe("DashboardCard", () => {
  it("renders DashboardCard", () => {
    const tree = renderer
      .create(
        <DashboardCard
          createdAt="2021-08-11T13:38:49.000Z"
          title="This is title"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
