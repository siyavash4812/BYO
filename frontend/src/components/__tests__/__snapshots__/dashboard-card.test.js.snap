// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardCard renders DashboardCard 1`] = `
.c1 {
  font-size: 1em;
  color: #001f3f;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 20px;
  border: 1px dotted peru;
  margin: 10px 0px;
  background-color: #DDDDDD;
}

<div
  className="c0"
  data-testid="dashboard-card"
>
  <span
    className="c1"
    size="S"
  >
    Title: 
    This is title
  </span>
  <span
    className="c1"
    size="S"
  >
    Created: 
    2021-08-11 14-38-49
  </span>
</div>
`;
